core/dependencies/yaml-cpp

```diff
@@ -1 +1 @@
-Subproject commit 95bb6a18f9523b46499e9b870ef4b8e4d8068fbb
+Subproject commit a49337024219f7b3778cbcbfc3e6d26e84f9408f
```

Comment:
Am I missing something? But I am not able to update submodule post merge to master.

I get the following on "git submodule update":
```
fatal: reference is not a tree: a49337024219f7b3778cbcbfc3e6d26e84f9408f
Unable to checkout 'a49337024219f7b3778cbcbfc3e6d26e84f9408f' in submodule path 'core/dependencies/yaml-cpp'
```